src/metabase/api/card.clj

```diff
@@ -16,20 +16,16 @@
             [metabase.email.messages :as messages]
```

Comment:
Not thing being removed here, just re-order stuff.

Reply:
I actually made changes to this file earlier but then revert it, sr if it's hard to review